projects/components/src/select/select.component.ts

```diff
@@ -90,13 +100,16 @@ export class SelectComponent<V> implements AfterContentInit, OnChanges {
   @Input()
   public highlightSelected: boolean = true;
 
+  @Input()
+  public multiSelectMode: boolean = false;
+
   @Output()
   public readonly selectedChange: EventEmitter<V> = new EventEmitter<V>();
 
   @ContentChildren(SelectOptionComponent)
   public items?: QueryList<SelectOptionComponent<V>>;
 
-  public selected$?: Observable<SelectOption<V> | undefined>;
+  public selected$?: Observable<SelectOption<V> | SelectOption<V>[] | undefined>;
```

Comment:
Cleaner to always work in arrays IMO

Reply:
It'll definitely make code a lot more cleaner.

Reply:
I think it makes sense to use union types so that existing consumers don't need to be modified.